build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -22,7 +22,7 @@ public class AzurePublisher
 
         public AzurePublisher()
         {
-            _connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING").Trim('"');
+            _connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING")?.Trim('"');
```

Comment:
fixed.

---
In reply to: [64611612](https://github.com/dotnet/cli/pull/3230#discussion_r64611612) [](ancestors = 64611612)